docs/modules/ROOT/partials/getting-started/ref-registry-artifact-states.adoc

```diff
@@ -0,0 +1,39 @@
+// Metadata created by nebel
+// ParentAssemblies: assemblies/getting-started/as_registry-reference.adoc
+
+[id="registry-artifact-states"]
+
+= {registry} artifact states
+These are the valid artifact states in {registry}:
+
+.{registry} artifact states
+[%header,cols=2*]
+|===
+|State
+|Description
+|`ENABLED`
+|Basic state, all operations available.
+|`DISABLED`
+|The artifact and it's metadata is viewable and searchable through the UI but it's content cannot be fetched by any client.
```

Comment:
Suggest "...its metadata is viewable and searchable using the web console, but its content cannot..."